ui/src/app/signed-in-app-routing.tsx

```diff
@@ -33,7 +34,7 @@ import {profileStore} from './utils/stores';
 
 const registrationGuard: Guard = {
   allowed: (): boolean => hasRegisteredAccess(profileStore.get().profile.accessTierShortNames),
-  redirectPath: '/'
+  redirectPath: '/data-access-requirements'
```

Comment:
currently the homepage has a special mode "workbenchAccessTasks=true" meaning "the user needs to complete the registration tasks".  This story (maybe this PR?) will eventually remove that mode!

Therefore: it will no longer be appropriate to route unregistered users to the homepage